src/validation.cpp

```diff
@@ -1216,8 +1216,15 @@ void InitScriptExecutionCache() {
 
 /**
  * Check whether all inputs of this transaction are valid (no double spends, scripts & sigs, amounts)
- * This does not modify the UTXO set. If pvChecks is not NULL, script checks are pushed onto it
- * instead of being performed inline.
+ * This does not modify the UTXO set.
+ *
+ * If pvChecks is not NULL, script checks are pushed onto it instead of being performed inline. Any
+ * script checks which are not necessary (eg due to script execution cache hits) are, obviously,
+ * not pushed onto pvChecks/run.
+ *
+ * Setting cacheSigStore/cacheFullScriptStore to true will remove elements from the corresponding cache
```

Comment:
In commit "Better document CheckInputs parameter meanings"

s/true/false I think. Thanks for the comments!